src/components/Dashboards/Employee/leave.balances.tsx

```diff
@@ -5,7 +5,10 @@ import {
   ArrowForwardOutlined,
   ArrowRightAltOutlined,
   ArrowRightOutlined,
```

Comment:
It's nice to remove all unused imports, variables and functions in all our scripts.